tests/transforms/test_broadcast_expand.py

```diff
@@ -260,43 +233,44 @@ def cost(*params):
         jac = t.jacobian(out, params)
         exp_jac = t.jacobian(exp, params)
 
-        if len(obs) > 1 and size == 1:
-            new_exp_jac = [qml.math.flatten(el) for el in exp_jac]
-            new_jac = [qml.math.flatten(el) for el in jac]
-            assert all(qml.math.allclose(_jac, e_jac) for _jac, e_jac in zip(new_jac, new_exp_jac))
```

Comment:
The extraction of the first entry of `out` in the `exp_fn`s takes care of this already. There is nothing to flatten.